users/updates/components/ockrg4eU33GP.php

```diff
@@ -8,7 +8,7 @@
 $countE=0;
 
 $db->query("ALTER TABLE users
-  MODIFY COLUMN password varchar(255) NULL");
+  ADD COLUMN password varchar(255) NULL");
```

Comment:
Original was `MODIFY`, why the `ADD`?

Reply:
Same reason as mentioned before. If you have a suggestion for keyword other than ADD i can test it

Reply:
The column already existed of this type. I can test it by changing the column type and then running the query to see if ADD is the right keyword to use here. What do you think professor?

Reply:
> The column already existed of this type. I can test it by changing the column type and then running the query to see if ADD is the right keyword to use here. What do you think professor?

I can test it tomorrow as today there is no electricity in the dormitory due to renovation or maintenance :(

Reply:
`ADD` is right when the column did not exist previously. When it's already there, you have to use `ALTER`.

Reply:
Ok I will change both of them to `ALTER` professor